docs/doc/14-sql-commands/00-ddl/50-udf/ddl-create-function.md

```diff
@@ -3,20 +3,38 @@ title: CREATE FUNCTION
 description:
   Create a new user-defined scalar function.
 ---
+import FunctionDescription from '@site/src/components/FunctionDescription';
 
+<FunctionDescription description="Introduced or updated: v1.2.116"/>
 
-## CREATE FUNCTION
-
-Creates a new UDF (user-defined function), the UDF can contain an SQL expression.
+Creates a user-defined function.
 
 ## Syntax
 
 ```sql
-CREATE FUNCTION [ IF NOT EXISTS ] <name> AS ([ argname ]) -> '<function_definition>'
+-- Create with lambda expression
+CREATE FUNCTION [IF NOT EXISTS] <function_name> AS (<input_param_names>) -> <lambda_expression> [DESC='<description>']
+
+-- Create with UDF server
```

Comment:
<img width="838" alt="image" src="https://github.com/datafuselabs/databend/assets/172204/df99dcd7-4701-49fb-9d3a-f270ce3de8f4">
It's better to make it pretty format and no need to scroll the tab.